Source/Core/VideoBackends/Vulkan/VKTexture.cpp

```diff
@@ -85,46 +84,28 @@ std::unique_ptr<VKTexture> VKTexture::Create(const TextureConfig& tex_config, st
                                   nullptr,
                                   VK_IMAGE_LAYOUT_UNDEFINED};
 
-  VkImage image = VK_NULL_HANDLE;
-  VkResult res = vkCreateImage(g_vulkan_context->GetDevice(), &image_info, nullptr, &image);
-  if (res != VK_SUCCESS)
-  {
-    LOG_VULKAN_ERROR(res, "vkCreateImage failed: ");
-    return nullptr;
-  }
-
-  // Allocate memory to back this texture, we want device local memory in this case
-  VkMemoryRequirements memory_requirements;
-  vkGetImageMemoryRequirements(g_vulkan_context->GetDevice(), image, &memory_requirements);
-
-  VkMemoryAllocateInfo memory_info = {
-      VK_STRUCTURE_TYPE_MEMORY_ALLOCATE_INFO, nullptr, memory_requirements.size,
-      g_vulkan_context
-          ->GetMemoryType(memory_requirements.memoryTypeBits, VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT,
-                          false)
-          .value_or(0)};
+  VmaAllocationCreateInfo alloc_create_info = {};
+  alloc_create_info.flags = 0;
+  alloc_create_info.usage = VMA_MEMORY_USAGE_UNKNOWN;
```

Comment:
Similar to how you handle priority, if this is a compute image or a render target then you'll want to do:

```
if (tex_config.IsComputeImage() || tex_config.IsRenderTarget())
  alloc_create_info.usage = VK_IMAGE_USAGE_SAMPLED_BIT | VK_IMAGE_USAGE_COLOR_ATTACHMENT_BIT;
else
  alloc_create_info.usage = VMA_MEMORY_USAGE_UNKNOWN;
```

(not sure if `UNKNOWN` is appropriate)

Reply:
You're confusing [VmaAllocationCreateInfo](https://gpuopen-librariesandsdks.github.io/VulkanMemoryAllocator/html/group__group__alloc.html#gaa5846affa1e9da3800e3e78fae2305cc) with VkImageCreateInfo. Those aren't valid values for VmaAllocationCreateInfo::usage.

Besides, we're already passing the VkImageCreateInfo to vmaCreateImage, so if VMA wants to get clever with that, it already has the necessary usages.

Reply:
I used `VMA_MEMORY_USAGE_GPU_ONLY` previously for textures.  But we might need to play with it.

Reply:
@K0bin - those are valid.  Look at [the docs](https://gpuopen-librariesandsdks.github.io/VulkanMemoryAllocator/html/usage_patterns.html).  They are showing examples for `VmaAllocationCreateInfo` which is what you use here.

Reply:
@K0bin - whoops, thanks updated!